docs/crate_universe.md

```diff
@@ -360,8 +360,7 @@ bazel run //3rdparty:crates_vendor
 Repinning dependencies is controlled by both the `CARGO_BAZEL_REPIN` environment variable or the `--repin`
 flag to the `crates_vendor` binary. To update dependencies, simply add the flag ro your `bazel run` invocation.
 
-```shell
-bazel run //3rdparty:crates_vendor -- --repin
+```shellbazel
```

Comment:
Was this intentional?

Reply:
Not intentional, sorry about that. Removed that change.